pages/_plugins/data_page_generator.rb

```diff
@@ -0,0 +1,136 @@
+# coding: utf-8
+# Generate pages from individual records in yml files
```

Comment:
Would you give me a high-level overview of the difference between page_gen and a Jekyll collection?

Reply:
Of course -- [page_gen](https://github.com/avillafiorita/jekyll-datapage_gen) takes a CSV or yaml file and generates one page per record. This plugin is solely for the glossary. There's some [custom code here taken from the previous PRA example](https://github.com/GSA/digitalgov-pra/commit/bf1f3fabaa7691d62bc7194b06587296a4b253fb).